exp/views/study.py

```diff
@@ -455,50 +455,47 @@ def get_context_data(self, **kwargs):
         """
         context = super(StudyDetailView, self).get_context_data(**kwargs)
 
-        study: Study = context["study"]
+        study = self.object
         admin_group = study.admin_group
-
-        context["triggers"] = get_permitted_triggers(
-            self, self.object.machine.get_triggers(self.object.state)
-        )
-        context["logs"] = self.study_logs
-        state = context["state"] = self.object.state
-        context["status_tooltip"] = STATUS_HELP_TEXT.get(state, state)
-        context["current_researchers"] = self.get_annotated_study_researchers()
-        context["users_result"] = self.search_researchers()
-        context["build_ui_tag"] = "success" if study.built else "warning"
-        context["state_ui_tag"] = STATE_UI_SIGNALS.get(study.state, "info")
-        context["search_query"] = self.request.GET.get("match", "")
-        context["name"] = self.request.GET.get("match", None)
-        context["multiple_admins"] = (
-            len(User.objects.filter(groups__name=admin_group.name)) > 1
-        )
-        context["study_admins"] = User.objects.filter(
-            groups__name=admin_group.name
-        ).values_list("id", flat=True)
```

Comment:
Looks like the `study_admins` context attribute wasn't included in your version. Just checking if this was an error or intentional?

Reply:
Good eye!  That value is not used in any template in our project.  

<img width="1552" alt="Screenshot 2023-08-25 at 9 50 14 AM" src="https://github.com/lookit/lookit-api/assets/44074998/367954c3-94fe-4e4d-be67-b7068011edf5">